src/uwtools/config/validator.py

```diff
@@ -19,6 +20,25 @@
 # Public functions
 
 
+def bundle(schema: dict) -> dict:
```

Comment:
This function recursively dereferences the schema, resulting in a complete, standalone schema. The JSON Schema maintainers refer to this process as [bundling](https://json-schema.org/blog/posts/bundling-json-schema-compound-documents).